setup/generate_coverage_report/test.sh

```diff
@@ -26,6 +26,7 @@ UPLOAD_URL=https://transfer.sh
 #export PACKIT_SOURCE_URL=https://github.com/ansasaki/keylime
 #export PACKIT_SOURCE_SHA=a79b05642bbe04af0ef0a356afd4f5af276898bb
 
+OMIT_FILES="--omit=/var/lib/keylime/secure/unzipped/*,/usr/local/lib/python3.10/site-packages/keylime-6.4.2-py3.10.egg/keylime/backport_dataclasses.py"
```

Comment:
Please, use wildcards (*) here, we won't stick to a certain keylime version forever.

Reply:
Moreover, this filter should probably apply also to installed distribution keylime which is not in `/usr/local/`. I think filter like `*/keylime/backport_dataclasses.py` might be enough.